tests/test_object_model/test_events.py

```diff
@@ -494,3 +494,87 @@ def test_block_discharge(test_db):
     assert (
         np.abs(event.dis_df[1][-1] - event.attrs.river[0].base_discharge.value) < 0.001
     )
+
+
+class Test_ReadCSV:
+    def write_dummy_csv(
+        self, path, headers: bool, datetime_format: str, num_columns: int = 1
+    ) -> pd.DataFrame:
+        size = 20
+        gen = np.random.default_rng()
+        time = pd.date_range(start="2021-01-01", periods=size, freq="H")
+        data = {f"data_{i}": gen.random(size) for i in range(num_columns)}
+
+        df = pd.DataFrame(
+            {
+                "time": time,
+                **data,
+            }
+        )
+        df.to_csv(path, index=False, header=headers, date_format=datetime_format)
+
+        df = df.set_index("time")
+        return df
+
+    def test_read_csv_ReadFileWithHeaders(self, tmp_path):
```

Comment:
please use either snake_case or CamelCase but a combination of the two is a bit strange